### PR TITLE
Move docs to git

### PR DESCRIPTION
**Pull request by heiglandreas**
This will modify the toolchain to pull in the documentation from git instead of SVN.

Language folders are available by now, but the rev-check process that is built via the doc-repo needs to be checked first!

### CODE REVIEW

**Comment by heiglandreas** [comment]
In a chat between Gabriel Caruso and Paul Dragoonis we decided that we might skip fixing the rev-check part to get the docs as soon as possible over to git as the leading system.

**Comment by heiglandreas** [comment]
Fit to be merged right now!

**Comment by nikic** [comment]
Without having access to the box, will there be some way to tell whether the docs have been built from git once this is merged?

File: update-docs-sources [hunk 1]
```diff
@@ -10,6 +10,13 @@ for i in phpdoc-all; do
 	svn up -q
 done
 
+langs=`/usr/bin/php -r 'include "/local/src/phpweb/include/languages.inc"; echo implode( " ", array_keys( $ACTIVE_ONLINE_LANGUAGES ) );'`;
+langs="base $langs"
+for i in $langs; do
+  cd /local/src/phpdoc-git/$i
+  git pull
```
Comment (nikic) [inline update-docs-sources:17]: Does `git pull` work correctly with force pushes? Possibly this should `git fetch origin && git checkout origin/master`?

Also, as written now this requires someone with access to the server to do the initial clone of the repos, right?

Comment (heiglandreas) [reply update-docs-sources:17]: That was always the case. but the repos are already existing and are already updated constantly on the machine (as this file is not ion sync between git and euk2. 

And yes: The current version of the file is doing a `git fetch origin && git reset --hard origin/master`. I will adapt the script accordingly

**Review by nikic — APPROVED** [review]

**Pull request closed without merging**